oap-server/server-core/src/main/java/org/apache/skywalking/oap/server/core/analysis/meter/function/MeterFunction.java

```diff
@@ -29,6 +30,7 @@
  */
 @Target(ElementType.TYPE)
 @Retention(RetentionPolicy.RUNTIME)
+@Inherited
```

Comment:
I think this will affect the class scan, such as MeterSystem, functionRegister could be wrong.